ruby/spec/cucumber/ci_environment/detect_ci_environment_spec.rb

```diff
@@ -3,15 +3,15 @@
 require 'cucumber/ci_environment'
 require 'json'
 
-describe 'detect_ci_environment' do
+describe Cucumber::CiEnvironment, '.detect_ci_environment' do
```

Comment:
this should be in 2 blocks. So the top level should say 
> describe Cucumber::CiEnvironment

and then underneath it it should have the 2nd block

Reply:
Do you mean refactor it to be ?

describe Cucumber::CiEnvironment do
  it 'detects CI environment' do
    <test code>
  end
end

Reply:
2nd line should be `describe '.detect_ci_environment'`